Contributing.md

```diff
@@ -4,6 +4,15 @@
 SPDX is always welcoming new contributors! The discussions are happening on the spdx-tech mailing list
 and during our weekly meetings. All the details are in: https://spdx.dev/participate/tech/
 
+## Reviewing the SPDX 3.0 Release Candidate
+SPDX recently released the first release candidate for the SPDX 3.0 model.  
+We expect more release candidates prior to general availability and would welcome review and feedback.
+
+This repository consists of markdown files describing the model classes, properties, and vocabularies.
```

Comment:
, which will be used to automatically create documentation, ontologies, and validation artifacts.